app/webrtc.go

```diff
@@ -48,6 +49,17 @@ func GetWebrtcToken(sessionId string) (string, *model.AppError) {
 		return "", model.NewAppError("WebRTC.getWebrtcToken", "api.webrtc.disabled.app_error", nil, "", http.StatusNotImplemented)
 	}
 
+	switch strings.ToLower(*utils.Cfg.WebrtcSettings.GatewayType) {
+	case "kopano-webmeetings":
+		return GetKopanoWebmeetingsWebrtcToken(sessionId)
+	case "janus":
```

Comment:
why keep this option if is going to fall through the default?

Reply:
No technical reason - it only to clarify in code that `janus` is the default.